tests/dotnet/UnitTests/BundleStructureTest.cs

```diff
@@ -57,6 +58,7 @@ void CheckAppBundleContents (ApplePlatform platform, string appPath, string[] ru
 				case "libmono-component-debugger.dylib":
 				case "libmono-component-diagnostics_tracing.dylib":
 				case "libmono-component-hot_reload.dylib":
+				case "libmono-component-marshal-ilgen.dylib":
```

Comment:
@dalexsoto @rolfbjarne I think this is probably the change (plus the `@(_MonoComponent)` above, we're hitting that here:

https://github.com/dotnet/maui/pull/9636#pullrequestreview-1084455437